Casks/k/keyguard.rb

```diff
@@ -0,0 +1,35 @@
+cask "keyguard" do
+  version "r20241002,1.6.0"
+  sha256 "3631112e36eb33d5b9c7d2019bd418650c777d98b1fe2af1f9797e47364bf0ef"
+
+  url "https://github.com/AChep/keyguard-app/releases/download/#{version.csv.first}/Keyguard-#{version.csv.second}.dmg"
+  name "Keyguard"
+  desc "Is an alternative client for the Bitwarden platform"
+  homepage "https://github.com/AChep/keyguard-app"
+
+  livecheck do
+    url :url
+    regex(%r{/r(\d+)/Keyguard[._-](\d+(?:\.\d+)*)\.dmg}i)
+    strategy :github_latest do |json, regex|
+      json["assets"]&.map do |asset|
+        match = asset["browser_download_url"]&.match(regex)
+        next if match.blank?
+
+        "r#{match[1]},#{match[2]}"
+      end
```

Comment:
```suggestion
    regex(%r{/r?(\d+(?:\.\d+)*)/Keyguard[._-](\d+(?:\.\d+)+)\.dmg}i)
    strategy :github_latest do |json, regex|
      json["assets"]&.map do |asset|
        match = asset["browser_download_url"]&.match(regex)
        next if match.blank?

        "#{match[2]},#{match[1]}"
      end
```

This should make the regex more robust against changes in the tag naming scheme.